File: api-angular/StatinaryProject/Model/Shop Information.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace StatinaryProject.Model
{
    public class Shop_Information
    {
        [Key]
        public int Shop_Id { get; set; }
     
        public string Shop_Name { get; set; }
        public string Address { get; set; }
      
        public string MobileNo { get; set; }
        public virtual ICollection<Product_Information>? Product_Information { get; set; }

    }


   
}

File: api-angular/Stationary_AngularF/StationaryProjectF/src/app/Model/Shop_Information.ts
export class Shop_Information {
  public Shop_Id?: number;

  public Shop_Name?: string;

  public Address?: string;

  public MobileNo?: string;

  constructor(
    _Shop_Id?: number,
    _Shop_Name?: string,
    _Address?: string,
    _MobileNo?: string
  ) {
    this.Shop_Id = _Shop_Id;
    this.Shop_Name = _Shop_Name;
    this.Address = _Address;
    this.MobileNo = _MobileNo;
  }
}
